package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "scrap",
+  "version": "0.0.0",
+  "private": true,
+  "scripts": {
+    "start": "nodemon ./bin/www"
+  },
+  "dependencies": {
+    "axios": "^0.19.2",
+    "cheerio": "^1.0.0-rc.3",
+    "colors": "^1.4.0",
+    "cookie-parser": "~1.4.4",
+    "debug": "~2.6.9",
+    "express": "~4.16.1",
+    "file-system": "^2.2.2",
+    "http-errors": "^1.6.3",
+    "json2csv": "^5.0.1",
```

Comment:
no need to complicate things. we don't need this. just need to write what is needed in program.

Reply:
json2csv, file-system remove